coscale-dev-env/azuredeploy.json

```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "coscaleKey": {
+      "type": "string",
+      "metadata": {
+        "description": "CoScale registration key."
+      }
+    },
+    "coscaleEmail": {
+      "type": "string",
+      "metadata": {
+        "description": "Email for the CoScale Super User."
+      }
+    },
+    "coscalePassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the CoScale Super User."
+      }
+    },
+    "publicDnsName": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Name for the Public IP used to access CoScale."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_DS3_v2",
+      "allowedValues": [
+        "Standard_DS3_v2",
+        "Standard_DS4_v2",
+        "Standard_DS5_v2"
+      ],
+      "metadata": {
+        "description": "CoScale Virtual Machine Size"
+      }
+    }
+  },
+  "variables": {
+    "scriptsBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/coscale-dev-env/",
```

Comment:
See https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc for how to reference artifacts